docs/source/demo.md

```diff
@@ -1,67 +1,101 @@
+http://<status-url>/status
 ```
 
-## 2. Set up your source streamer  and / or capture software.
+## 1. Create your source streamer and / or capture software.
 
 There are many tools for this. Some common ones include:
 
+* FFmpeg from a mac laptop
+  * ```
+    ffmpeg -f avfoundation -framerate 60 -pixel_format uyvy422 -i "0:0" -vcodec libx264 -acodec aac -b 500k -x264-params keyint=120:min-keyint=120 -f flv rtmp://<ingest-url>/?manifestID=stream1
+    ```
+* FFmpeg for a video file
+  * ```
+    ffmpeg -re -i <input-video-file> -c:a copy -c:v copy -f flv rtmp://<ingest-url>/?manifestID=stream1
+    ```
 * [OBS](https://obsproject.com/) for desktops and laptops
   * Refer to our [guide](https://livepeer.readthedocs.io/en/latest/broadcasting.html#broadcasting-to-a-local-node-using-obs) on using OBS with Livepeer
 * [ManyCam](https://manycam.com) for mobile
-* FFmpeg for the command line
-  * ```
-    ffmpeg -re -i <input-video-file> -c:a copy -c:v copy -f flv rtmp://ingest-host.livepeer.org/stream/customerName_streamName.m3u8
-    ```
+
 * Re-stream or proxy RTMP traffic from your existing network into Livepeer
 
-## 3. Configure your player with the Livepeer output URL
+## 2. Get status about your source stream
 
-An easy option is to use the [Livepeer player](http://media.livepeer.org/)
+Using the status endpoint, you can get status of all of the streams on a Livepeer node.  For example:
+```
+curl http://<status-url>/status | jq
+```
+
+You should see something like
+```
+{
+  "Manifests": {
+    "stream1": "#EXTM3U\n#EXT-X-VERSION:3\n#EXT-X-STREAM-INF:PROGRAM-ID=0,BANDWIDTH=4000000,RESOLUTION=1280x720\njohn/source.m3u8\n#EXT-X-STREAM-INF:PROGRAM-ID=0,BANDWIDTH=1200000,RESOLUTION=640x360\njohn/P360p30fps16x9.m3u8\n#EXT-X-STREAM-INF:PROGRAM-ID=0,BANDWIDTH=600000,RESOLUTION=426x240\njohn/P240p30fps16x9.m3u8\n"
+  },
+  "Version": "0.3.3-0.3.2-42-g063c2f9"
+}
+```
+If you see an error like `-bash: jq: command not found`, use 
+```
+curl http://<status-url>/status
+```
+
+## 3. View your stream
 
-## 4. Start streaming, have fun!
+An easy option is to use the [Livepeer player](http://media.livepeer.org/).  Simply put the stream output URL there to view your stream.  For example: 
+```
+http://<streaming-url>/stream/stream1.m3u8
+```
 
 ## FAQs
 
-**Q:** How can I obtain an ingest URL?\
+**Q: How can I obtain an ingest URL?**\
 **A:** Contact beta@livepeer.org with the title "Ingest URL Provision Request".
 
-**Q:** How much does the transcoding service cost?\
-**A:** For a limited period of March 2019 to September 2019, the transcoding service is completely free of charge.  In the future, Livepeer will be 10x to 50x cheaper than current cloud transcoding APIs.
+**Q: How much does the transcoding service cost?**\
+**A:** For a limited period of March 2019 to September 2019, the transcoding service is completely free of charge.  In the future, Livepeer will be less than $0.05 per hour per stream.
```

Comment:
We don't really know what prices the network will settle with; can we keep this promise?

Reply:
I understand the hesitance, but I'd like to provide a number to make it concrete.  I don't see this as a definitive promise to the world. (Especially given it's on a documentation page and not on our homepage)